spec/sidekiq/batch_spec.rb

```diff
@@ -170,7 +170,7 @@ def was_performed; end
         Sidekiq::Batch.process_failed_job(bid, 'failed-job-id')
         Sidekiq::Batch.process_failed_job(bid, failed_jid)
         failed = Sidekiq.redis { |r| r.smembers("BID-#{bid}-failed") }
-        expect(failed).to eq(['xxx', 'failed-job-id'])
+        expect(failed).to eq(['failed-job-id', 'xxx'])
```

Comment:
According to the [SMEMBERS](https://redis.io/commands/smembers/) redis documentation, this order is to be expected. The `Sidekiq::Batch.process_failed_job` method calls `SADD` to store the failed jobs. `SMEMBERS` returns these in the order in which they were added. Or perhaps better to to change expected value to a Ruby `Set` because in a set the order is irrelevant.